lib/workers/repository/package-files.ts

```diff
@@ -0,0 +1,64 @@
+import type { RenovateConfig } from '../../config/types';
+import { logger } from '../../logger';
+import type { PackageFile } from '../../modules/manager/types';
+
+export class PackageFiles {
+  private static data: Map<string, Record<string, PackageFile[]> | null> =
+    new Map<string, Record<string, PackageFile[]>>();
+
+  public static add(
+    baseBranch: string,
+    packageFiles: Record<string, PackageFile[]> | null
+  ): void {
+    logger.debug(
+      `PackageFiles.add() - Package file saved for branch=${baseBranch}`
+    );
```

Comment:
```suggestion
    logger.debug( { baseBranch },
      `PackageFiles.add() - Package file saved for branch`
    );
```

Use logger meta instead?